tests/functional/install/destructive_LVM_RAID_combination.bats

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+wait_for_raid_to_be_active() {
+    while :
+    do
+        if [[ $(cat /proc/mdstat | grep -i "resync|delayed") ]]; then
```

Comment:
Is something in BATS already watching for long-running tests?
Or does this loop need to count the number of sleeps and fail after a determined amount of time?

Reply:
Thats a good idea. Maybe i should sleep a while before timing out.